src/ensstat.fd/ensstat.f90

```diff
@@ -320,13 +339,13 @@ program ens_avgspr_g2
         ! get the number of bits
         ! gfldo%idrtmpl(3) : GRIB2 DRT 5.40 decimal scale factor
 
-        write(6,*) 'gfldo%idrtmpl(3)=',gfldo%idrtmpl(3)
+        ! write(6,*) 'gfldo%idrtmpl(3)=',gfldo%idrtmpl(3)
 
-        call gtbits(0,gfldo%idrtmpl(3),maxgrd,0,ens_avg,gmin,gmax,nbit)
+        !call gtbits(0,gfldo%idrtmpl(3),maxgrd,0,ens_avg,gmin,gmax,nbit)
```

Comment:
```suggestion
```
The file `gtbits.f90` is removed from compilation.  It should also be removed from the repository.
This call is no longer going to be valid. Leaving it commented is not advisable.  Suggest removing it.